### PR TITLE
Vet clinic database: create animals table

### PR DESCRIPTION
**Pull request by arnrdn**
Hello!

In this PR I:
1. Created a database called 'vet_clinic'
2. Created a table called 'animals'
3. Added data to the table
4. Created queries to filter the data in the table and get the information needed

Thank you for your review!

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT name FROM animals WHERE name LIKE '%mon';
```
Comment (julie-ify) [inline queries.sql]: - [ ] This query returns the names of all animals that its name ends in "mon". However, we are asked to find all animals `not just the names of the animals but everything about the animals` that its name ends in "mon". A good query would be `SELECT * FROM animals WHERE name LIKE '%mon;`

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+SELECT date_of_birth FROM animals WHERE date_of_birth BETWEEN '2016/01/01' AND '2019/12/31';
```
Comment (julie-ify) [inline queries.sql]: - [ ]  This query returns the `date_of_birth` of all animals that  born between 2016 and 2019. However, we are asked to list the `name` of all animals born between 2016 and 2019. A good query would be `SELECT name FROM animals WHERE date_of_birth BETWEEN '2016/01/01' AND '2019/12/31';`

File: queries.sql [hunk 3]
```diff
@@ -1,3 +1,10 @@
...
+SELECT escape_attempts FROM animals WHERE neutered = false AND escape_attempts < 3;
```
Comment (julie-ify) [inline queries.sql]: - [ ] This query returns the `escape_attempts` of all animals that are not neutered and have less than 3 escape attempts. However, we are asked to list the name of all animals that are neutered and have less than 3 escape attempts. A good query would be `SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;`

File: queries.sql [hunk 4]
```diff
@@ -1,3 +1,10 @@
...
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT * FROM animals WHERE neutered = true;
+SELECT * FROM animals WHERE name != 'Gabumon';
+SELECT * FROM animals WHERE weight_kg >= 10.4 OR weight_kg <= 17.3;
```
Comment (julie-ify) [inline queries.sql]: - [ ] Line 10 query returns all animals with a weight between 10.4kg or 17.3kg. However, we are asked to find all animals with a weight between 10.4kg and 17.3kg. A good query would be `SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
`

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @rdnrn,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mmsesay — APPROVED** [review]
Hi @rdnrn, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**